fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -308,15 +277,15 @@ public static void stopTiming(String timePhaseName) {
             LOG.warn("Tried to stop non-existent phase: {}", timePhaseName);
             return;
         }
-        timePhase.stop();
+        timePhase.close();
```

Comment:
If we make "stop" convert to doing the registering, calling "close" is likely not needed, but "stop" will be.